README.md

```diff
@@ -11,13 +11,13 @@ for the TLS (formerly SSL), DTLS and QUIC (currently client side only)
 protocols.
 
 The protocol implementations are based on a full-strength general purpose
-cryptographic library, which can also be used stand-alone. Also included is a
+cryptographic library, which can also be used standalone. Also included is a
 cryptographic module validated to conform with FIPS standards.
 
 OpenSSL is descended from the SSLeay library developed by Eric A. Young
 and Tim J. Hudson.
 
-The official Home Page of the OpenSSL Project is [www.openssl.org].
+The official Homepage of the OpenSSL Project is [www.openssl.org].
```

Comment:
Since it's not a proper noun, we don't really need the capitalised form of `homepage`